pubsub/subscriptions/main.go

```diff
@@ -89,12 +89,9 @@ func pullMsgs(client *pubsub.Client, name string, topic *pubsub.Topic) error {
 
 	// publish 10 messages on the topic.
 	for i := 0; i < 10; i++ {
-		_, err := topic.Publish(ctx, &pubsub.Message{
+		topic.Publish(ctx, &pubsub.Message{
```

Comment:
Done, but it is awkward to do it in the same block. The developer would question why there is no blocking publish and what the merits of non-blocking publish has here. It is not quite a good experience having to write all the boilerplate to achieve a simple blocking publish.